.github/workflows/publish.yml

```diff
@@ -8,9 +8,9 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Set up Python 3.7
-        uses: actions/setup-python@v4
+        uses: actions/setup-python@v5
         with:
           python-version: 3.7
```

Comment:
I also think we should switch to a more recent python version here, but I did not want to change this without asking, as I can not test it 🤔